.github/workflows/release.yml

```diff
@@ -0,0 +1,22 @@
+name: release
+on:
+  push:
+    branches:
+      - main
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v4
+        with:
+          persist-credentials: false # to disable default GITHUB_TOKEN
+      
+      - name: Semantic Release
+        uses: cycjimmy/semantic-release-action@v4
+        env:
+          GITHUB_TOKEN: ${{ secrets.PAT }}
```

Comment:
I don't think this is needed. The github release plugin works with the default token afk